sympy/functions/combinatorial/numbers.py

```diff
@@ -1861,12 +1861,19 @@ def stirling(n, k, d=None, kind=2, signed=False):
 def _nT(n, k):
     """Return the partitions of ``n`` items into ``k`` parts. This
     is used by ``nT`` for the case when ``n`` is an integer."""
-    if 2*k >= n:
-        return partition(n - k)
-    if k == 0:
+    if k > n:
         return 0
-    if k == 1:
+    if k == n or k == 1:
```

Comment:
How should `k = n < 0` be handled?

Reply:
No problem now as `k >= 0`.

Reply:
n < 0, by definition of  counting sets of nonnegative integers, would be 0, wouldn't it?

Reply:
n < 0, by definition of  counting sets of nonnegative integers, would be 0, wouldn't it?